### ISSUE TITLE
Atherys in-game lore time

### ISSUE THREAD

**Issue by ghost** [Atherys-Horizons/AtherysRoleplay#3]
This plugin allows you to create a lore time system that actually works in game. This plugin can expand the length of days and nights, to suit your Lore Calendar.

Link to the plugin: https://github.com/AaronFoley/ItsAboutTime

**Comment by HaedHutner** [comment]
The plugin in question was written for Spigot and is therefore incompatible with our plugins.

**Comment by fingoltin** [comment]
Yeah he knows; he's just suggesting the idea which I will probably add at some point.
